pkg/tests/handlers/accounts_test.go

```diff
@@ -0,0 +1,100 @@
+package handlers
+
+import (
+	"context"
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/cloud-bulldozer/ocm-api-load/pkg/logging"
+	"github.com/cloud-bulldozer/ocm-api-load/pkg/types"
+)
+
+func Test_clusterAuthorizationsBody(t *testing.T) {
+	type args struct {
+		ctx       context.Context
+		clusterID string
+		options   *types.TestOptions
+	}
+	logger, _ := logging.NewGoLoggerBuilder().Build()
+	tests := []struct {
+		name string
+		args args
+	}{
+
+		{"randomize-1", args{context.TODO(), "clusterID-1", &types.TestOptions{Logger: logger}}},
+		{"randomize-2", args{context.TODO(), "clusterID-2", &types.TestOptions{Logger: logger}}},
+		{"randomize-3", args{context.TODO(), "clusterID-3", &types.TestOptions{Logger: logger}}},
+		{"randomize-4", args{context.TODO(), "clusterID-4", &types.TestOptions{Logger: logger}}},
+		{"randomize-5", args{context.TODO(), "clusterID-5", &types.TestOptions{Logger: logger}}},
+		{"randomize-6", args{context.TODO(), "clusterID-6", &types.TestOptions{Logger: logger}}},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := clusterAuthorizationsBody(tt.args.ctx, tt.args.clusterID, tt.args.options)
+			if got == nil {
+				t.Errorf("clusterAuthorizationsBody() = %v", got)
+			} else {
+				if !strings.Contains(string(got), tt.args.clusterID) {
```

Comment:
got should be already a string no need to cast it to string with `string(got)`

Reply:
I think it is of type byte, as clusterAuthorizationsBody returns []byte

Reply:
True, my bad.